flake8_black.py

```diff
@@ -98,6 +106,26 @@ def _file_mode(self):
             assert "got an unexpected keyword argument" in str(e), e
             return None
 
+    @classmethod
+    def add_options(cls, parser):
+        """Adding flake8-black-config option."""
+        parser.add_option(
+            "--flake8-black-config",
+            default=None,
+            action="store",
+            type="string",
+            parse_from_config=True,
+            help=(
+                "A path to the global configuration file for ``black``. This file will"
```

Comment:
How about something shorter:

"""python
help="Path to black configuration file (by default looks for pyproject.toml)"
"""

Reply:
Tbh it sounds a bit confusing to me, bu I'd be fine with: `Path to black configuration file (overrides the default pyproject.toml)`